Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM ubuntu:trusty
+FROM debian:7.8
```

Comment:
I prefer using `debian:7` or `debian:wheezy` to using pointing release like `debian:7.8`. It makes following the latest stable release easy to us.